pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2373,7 +2373,7 @@ func (c *Cloud) GetVolumeLabels(volumeName KubernetesVolumeID) (map[string]strin
 	labels := make(map[string]string)
 	az := aws.StringValue(info.AvailabilityZone)
 	if az == "" {
-		return nil, fmt.Errorf("volume did not have AZ information: %q", info.VolumeId)
+		return nil, fmt.Errorf("volume did not have AZ information: %s", *info.VolumeId)
```

Comment:
Is this related to the patch?

Reply:
Build of the tests fails due to more modern checks.

Reply:
Ok makes sense